packages/uikit/src/hooks/browser/useRecommendations.ts

```diff
@@ -0,0 +1,41 @@
+import { useUserCountry } from '../../state/country';
+import { useQuery } from '@tanstack/react-query';
+import { QueryKey } from '../../libs/queryKey';
+
+export interface CarouselApp extends PromotedApp {
+    poster: string;
+}
+
+export interface PromotedApp {
+    name: string;
+    description: string;
+    icon: string;
+    url: string;
+    textColor?: string;
+}
+
+export interface PromotionCategory {
+    id: string;
+    title: string;
+    apps: PromotedApp[];
+}
+
+export interface Recommendations {
+    categories: PromotionCategory[];
+    apps: CarouselApp[];
+}
+
+export function useRecommendations() {
+    const country = useUserCountry();
+    const lang = country.data || 'en';
+
+    return useQuery<Recommendations, Error>([QueryKey.featuredRecommendations, lang], async () => {
```

Comment:
Could you please add `track` - global query parameter for tonenponig class? the same as lang and build
Please define `desktop` and `extension` value for different applications in `App.tsx`

`track=desktop` for desktop and web and twa
`track=extension` for extension

Reply:
done